internal/agent/install.go

```diff
@@ -331,3 +339,14 @@ func prepareConfiguration(ctx context.Context, source string) (string, error) {
 
 	return f.Name(), nil
 }
+
+func generateInstallConfForCLIArgs(source string) string {
```

Comment:
it's hard to tell for me what kind of sources I can pass here? but down in the generated response it seems it is a URL, it's a minor detail but maybe we can extend it with `sourceURL` or such? then it also helps when writing a caller with the IDE

Reply:
renamed this to `sourceImageURL` like I did for the rest of the functions. At least it's more clear that it is the same value we are passing around.